projects/vdk-plugins/vdk-ipython-ext/setup.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import pathlib
+
+import setuptools
+
+"""
+Builds a package with the help of setuptools in order for this package to be imported in other projects
+"""
+
+__version__ = "0.1.0"
+
+setuptools.setup(
+    name="vdk-ipython-ext",
+    version=__version__,
+    url="https://github.com/vmware/versatile-data-kit",
+    description="Ipython extension for VDK",
+    long_description=pathlib.Path("README.md").read_text(),
+    long_description_content_type="text/markdown",
+    install_requires=["vdk-core"],
+    package_dir={"": "src"},
+    packages=setuptools.find_namespace_packages(where="src"),
+    # This is the only vdk plugin specifc part
+    # Define entry point called "vdk.plugin.run" with name of plugin and module to act as entry point.
+    entry_points={"vdk.plugin.run": ["ipython-ext = vdk_ipython_ext"]},
+    classifiers=[
+        "Development Status :: 4 - Beta",
```

Comment:
It should be pre-alpha or alpha at most at the current stage - https://github.com/vmware/versatile-data-kit/wiki/Coding-Standard#development-status